setup.md

```diff
@@ -41,7 +41,7 @@ Install the default version available from your distribution. For example, on Ub
 `sudo apt install default-jre`.
 * OpenRefine does not support Internet Explorer. Please use [Firefox](https://www.mozilla.org/firefox/new/), 
 [Chrome](https://www.google.com/chrome/) or [Safari](https://www.apple.com/safari/) instead.
-
+* For additional, system-speficic guidance on installing and running OpenRefine, see the "Running OpenRefine" section of the OpenRefine Documentation page, available at: <https://docs.openrefine.org/manual/running>.
```

Comment:
Shall we _merge_ this with the `Installation Instructions` sentence above?

For example, changing line 23 to:

> Please follow OpenRefine's manual to [install](https://docs.openrefine.org/manual/installing) and [run](https://docs.openrefine.org/manual/running) it.

I'm concerned that _splitting_ these instructions with 20 lines of notes in between reduces the utility of this page.

The `Notes:` section should IMHO rather _highlight_ a few common install & running problems, which it does best when it is one coherent block _after_ the links to the manual.

Reply:
Thank you @saragon02 for the suggestion. I will add Katrin's revison to the updated version of this page about line 23.